Sources/ClientRuntime/Config/DefaultHttpClientConfiguration.swift

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+public protocol DefaultHttpClientConfiguration: ClientConfiguration {
+
+    /// The HTTP client to be used for SDK HTTP requests.
+    ///
+    /// If none is provided, AWS SDK for Swift selects its own HTTP client for use:
+    /// - On Mac and Linux, a AWS-provided HTTP client is used for the best stability and performance with heavy AWS workloads.
```

Comment:
nit: please change this comment string, the AWS-provided HTTP client is only used on linux, on mac we are using the URLSession-based client